server/api/api/endpoints/background_tasks.py

```diff
@@ -197,17 +196,25 @@ async def list_internal_background_tasks(
 
 
 async def _authorize_get_background_task_request(
-    name: str, auth_info: mlrun.common.schemas.AuthInfo
+    background_task: mlrun.common.schemas.BackgroundTask,
+    auth_info: mlrun.common.schemas.AuthInfo,
 ):
-    # Since there's no not-found option on get_background_task - we authorize before getting (unlike other get endpoint)
-    # In Iguazio 3.2 the manifest doesn't support the global background task resource - therefore we have to just omit
-    # authorization
-    # we also skip Iguazio 3.6 for now, until it will add support for it (still in development)
-    igz_version = mlrun.mlconf.get_parsed_igz_version()
-    if igz_version and igz_version >= semver.VersionInfo.parse("3.7.0-b1"):
-        await server.api.utils.auth.verifier.AuthVerifier().query_resource_permissions(
-            mlrun.common.schemas.AuthorizationResourceTypes.background_task,
-            name,
+    # Iguazio manifest doesn't support the global background task resource - therefore if the background task has a
+    # project (e.g. delete project), we can authorize on the project
+    if background_task.metadata.project:
+        return await server.api.utils.auth.verifier.AuthVerifier().query_resource_permissions(
+            mlrun.common.schemas.AuthorizationResourceTypes.project,
+            background_task.metadata.project,
             mlrun.common.schemas.AuthorizationAction.read,
             auth_info,
         )
+
+    # If there is no project we have to just omit authorization
```

Comment:
why this is here?

Reply:
This needs to be supported one day so I would keep the comment to have a reference for it